### ISSUE TITLE
Button press not triggering in 7.0

### ISSUE THREAD

**Issue by rapamatic** [FlyingDiver/Indigo-radiora2#4]
I have several triggers based on button presses, that had been working in the prior version but aren't working now in 7.0.0 -- I will try to see if I can troubleshoot, but just curious if you changed any of the code around button press triggers - I see the button presses in the debug log, but the triggers aren't getting executed....

**Comment by FlyingDiver** [comment]
There were some changes to enable multiple triggers off the same button press event, but I don't think it would have broken anything.  There were also some changes to avoid a crash when a button was set up improperly.  Post the log section and I'll take a look.

**Comment by rapamatic** [comment]
I see these errors when I restart the plugin

```
   Reloading plugin "Lutron RRA2/Caséta 7.0.0"
   Stopping plugin "Lutron RRA2/Caséta 7.0.0" (pid 58643)
   Lutron RRA2/Caséta Debug       Removing Trigger Back Door All Off (1619281238)
   Lutron RRA2/Caséta Error       exception in triggerStopProcessing(Back Door All Off): 
   Lutron RRA2/Caséta Debug       Removing Trigger Back Door Off Double Tap (1772007950)
   Lutron RRA2/Caséta Error       exception in triggerStopProcessing(Back Door Off Double Tap): 
   Lutron RRA2/Caséta Debug       Removing Trigger LR Bright Double Tap (779933855)
   Lutron RRA2/Caséta Error       exception in triggerStopProcessing(LR Bright Double Tap): 
   Lutron RRA2/Caséta Debug       Removing Trigger LR Off Double Tap (239839145)
   Lutron RRA2/Caséta Error       exception in triggerStopProcessing(LR Off Double Tap): 
   Lutron RRA2/Caséta Debug       Removing Trigger Master Door Off Double Tap (957356661)
   Lutron RRA2/Caséta Error       exception in triggerStopProcessing(Master Door Off Double Tap): 
```

In the logs, I see that its checking for a trigger (in this case, Device 68,1.. which has a trigger setup, but no messages beyond that

```
   Nov 19, 2016, 9:23:49 PM
   Lutron RRA2/Caséta Debug       Received a Device message: ~DEVICE,68,1,3
   Lutron RRA2/Caséta Debug       Received a keypad button/LED status message: ~DEVICE,68,1,3
   Lutron RRA2/Caséta Debug       Received a Keypad Button press message, checking triggers: ~DEVICE,68,1,3
   Lutron RRA2/Caséta Debug       Received an Output message: ~OUTPUT,5,1,85.00
   Lutron RRA2/Caséta             Received: Dimmer Living Room - East Overhead Cans level set to 85.00
   Lutron RRA2/Caséta Debug       Received an Output message: ~OUTPUT,5,29,0
   Lutron RRA2/Caséta Debug       Received an Output message: ~OUTPUT,5,30,1,85.00
   Lutron RRA2/Caséta Debug       Received an Output message: ~OUTPUT,29,1,85.00
   Lutron RRA2/Caséta             Received: Dimmer Living Room - Middle Overhead Cans level set to 85.00
   Lutron RRA2/Caséta Debug       Received an Output message: ~OUTPUT,29,29,0
   Lutron RRA2/Caséta Debug       Received an Output message: ~OUTPUT,29,30,1,85.00
   Lutron RRA2/Caséta Debug       Received an Output message: ~OUTPUT,30,1,85.00
   Lutron RRA2/Caséta             Received: Dimmer Living Room - West Overhead Cans level set to 85.00
   Lutron RRA2/Caséta Debug       Received an Output message: ~OUTPUT,30,29,0
   Lutron RRA2/Caséta Debug       Received an Output message: ~OUTPUT,30,30,1,85.00
   Lutron RRA2/Caséta Debug       Received a Device message: ~DEVICE,68,81,9,1
   Lutron RRA2/Caséta Debug       Received a keypad button/LED status message: ~DEVICE,68,81,9,1
   Lutron RRA2/Caséta Debug       Received a Device message: ~DEVICE,68,1,3
   Lutron RRA2/Caséta Debug       Received a keypad button/LED status message: ~DEVICE,68,1,3
   Lutron RRA2/Caséta Debug       Received a Keypad Button press message, checking triggers: ~DEVICE,68,1,3
   Lutron RRA2/Caséta Debug       Received a Device message: ~DEVICE,68,1,3
   Lutron RRA2/Caséta Debug       Received a keypad button/LED status message: ~DEVICE,68,1,3
   Lutron RRA2/Caséta Debug       Received a Keypad Button press message, checking triggers: ~DEVICE,68,1,3
   Lutron RRA2/Caséta Debug       Received a Device message: ~DEVICE,68,1,3
   Lutron RRA2/Caséta Debug       Received a keypad button/LED status message: ~DEVICE,68,1,3
   Lutron RRA2/Caséta Debug       Received a Keypad Button press message, checking triggers: ~DEVICE,68,1,3
```

Here's a pic of my trigger:

<img width="890" alt="screen shot 2016-11-19 at 9 25 29 pm" src="https://cloud.githubusercontent.com/assets/967675/20460113/cdda8e16-ae9e-11e6-9bfe-c09ab6c990cf.png">

**Comment by FlyingDiver** [comment]
Should be fixed in 7.0.1

**Comment by rapamatic** [comment]
Working perfect for me now. Thanks!
